### PR TITLE
Generate Api Documentation with PostMan

### PR DESCRIPTION
**Pull request by julieg18**
#### What does this PR do?
- generates API documentation with postman
- edits `docs/api.js` to host API documentation with swagger
- edits the `completeTopic` function in `src/app/profiency/controller.js`
- adds env BASE_URL variable
- added `docs` to `.codeclimate.yml`
#### Description of Task to be completed?
We need documentation for our API.
#### How should this be manually tested?
* run `npm run dev`
* go to `api/v1/docs`
#### What are the relevant pivotal tracker stories?
[#167066158](https://www.pivotaltracker.com/story/show/)

### CODE REVIEW

File: docs/api.js [hunk 1]
```diff
@@ -2,21 +2,23 @@ export default {
   swagger: '2.0',
   info: {
     version: '1.0',
-    title: 'NewDev Api Documentation',
-    description: 'NewDev Api Documentation',
+    title: 'NewDev API Docs',
+    description: 'A documentation for our NewDev API.',
```
Comment (ebenezerdon) [inline docs/api.js]: Could we change the description to `NewDev Api Documentation`?

File: docs/api.js [hunk 2]
```diff
@@ -41,44 +43,800 @@ export default {
...
+            description: '',
+            headers: {},
+          },
+        },
+        security: [],
+      },
+    },
+    '/technologies/javascript': {
+      get: {
+        description: 'Get a technology.',
```
Comment (ebenezerdon) [inline docs/api.js]: Could we have this description as `Get a single technology`?

File: src/app/proficiency/controller.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const completeTopic = async (req, res) => {
       })
     : await Proficiency.create({
         userId: userId.trim(),
-        topicId: topicId.trim(),
+        topicId: topic.dataValues.id,
```
Comment (ebenezerdon) [inline src/app/proficiency/controller.js]: In your commit message, could you make this change more specific than `edit completeTopic function`?

File: docs/api.js (hunk 1 above)
Comment (julieg18) [reply docs/api.js]: Sure!

File: docs/api.js (hunk 2 above)
Comment (julieg18) [reply docs/api.js]: Of course

File: src/app/proficiency/controller.js (hunk 3 above)
Comment (julieg18) [reply src/app/proficiency/controller.js]: 👍

**Pull request merged**